doc/templates/dnf5-plugin/template_cmd_plugin.cpp

```diff
@@ -0,0 +1,77 @@
+#include "template.hpp"
+
+#include <dnf5/iplugin.hpp>
+
+using namespace dnf5;
+
+namespace {
+
+/* Enter the name of your new plugin. */
+constexpr const char * PLUGIN_NAME{"template"};
+
+/* Fill in the version of your plugin. */
```

Comment:
Would be possible to write why this is important, how it is used or whether is it mandatory?